nuget/src/it/test-nuget-common-flow.groovy

```diff
@@ -32,15 +32,15 @@ runCommand(targetPath, String.format(
     configPath))
 
 output = runCommand(targetPath, String.format(
-    "$nugetExec list Org.Carlspring -ConfigFile %s",
-    configPath))
+    "$nugetExec list Org.Carlspring -Source %s",
+    storageUrl))
```

Comment:
This should probably be a separate command? I assume ` "$nugetExec list Org.Carlspring -ConfigFile %s"` is making the listing using credentials? Also, could you pleas check if `nuget` is not automatically picking up the `NuGet.config` file? (this file is created a few lines before this)

Reply:
Changed to separate command.
Tested it with only `"$nugetExec list Org.Carlspring"` it was failing with reason 
**Unable to load the service index for source https://api.nuget.org/v3/index.json**
Hence we can say, it is not picking `NuGet.config`